manifests/params.pp

```diff
@@ -271,6 +271,7 @@
     'password'        => '',
     'dialect'         => '',
     'enable_h2_logs'  => 'on',
+    'maxactive'       => '50',
```

Comment:
An undef value is probably a "safer" default: unless a value is explicitely set, do not write a specific config.